### ISSUE TITLE
Undefined 'explicit_podcast' in feed description

### ISSUE THREAD

**Issue by RemySphere** [PodcastGenerator/PodcastGenerator#192]
Hello,
My podcast cannot be published to iTunes because it fails to populate the `<itunes:explicit>` tag correctly.
Relevant generated feed.xml snippet:
```
<itunes:owner>
    <itunes:name>Title</itunes:name>
    <itunes:email>dummy@dummy.com</itunes:email>
</itunes:owner>
<itunes:explicit/>
```

Apache's error.log:
```
Undefined index: explicit_podcast in /var/www/html/core/feed_generator.php on line 44, referer: http://mydomain.com/admin/podcast_details.php
```

**Comment by RemySphere** [comment]
Huh, if I guessed correctly, the generated config.php should have `$explicit_podcast` defined, but mine has `$explicit` instead...

**Comment by emilengler** [comment]
$explicit is wrong.
Change it to `$explicit_podcast` and set it to yes or no

**Comment by RemySphere** [comment]
It is how it gets generated. Even if I change it manually, tweaking the options and saving again overwrites my `$explicit_podcast` to `$explicit`

**Comment by emilengler** [comment]
Have you used Podcast Generator 2.7 before upgrading or were you still on 2.6?

**Comment by RemySphere** [comment]
Sorry I should have mentioned - it's a fresh install of the latest version available to download : PodcastGenerator 3.0.

**Comment by alwa** [comment]
I can confirm that config.php had the incorrect variable name $explicit after upgrading from 2.7
Changing to $explicit_podcast solved my problem
